front/hooks/useFileUploaderService.ts

```diff
@@ -106,20 +106,37 @@ export function useFileUploaderService({
   const processSelectedFiles = (
     selectedFiles: File[]
   ): Result<FileBlob, FileBlobUploadError>[] => {
-    return selectedFiles.map((file) => {
-      const contentType = getMimeTypeFromFile(file);
-      if (!isSupportedFileContentType(contentType)) {
-        return new Err(
-          new FileBlobUploadError(
-            "file_type_not_supported",
-            file,
-            `File "${file.name}" is not supported.`
-          )
-        );
-      }
+    return selectedFiles.reduce(
+      (acc, file) => {
+        if (fileBlobs.some((f) => f.id === file.name)) {
+          sendNotification({
```

Comment:
:droplet: 
should we send a notif to the user on this? They might misunderstand if framed as error

Reply:
actually :coin: is better than :droplet: